gridstatus/caiso.py

```diff
@@ -1460,19 +1460,19 @@ def _make_timestamp(time_str, today, timezone="US/Pacific"):
     return ts
 
 
-def _get_historical(file, date, verbose=False):
-    try:
-        date_str = date.strftime("%Y%m%d")
-        url = _HISTORY_BASE + "/%s/%s.csv" % (date_str, file)
-        msg = f"Fetching URL: {url}"
-        log(msg, verbose)
-    except Exception:
-        # fallback if today and no historical file yet
-        if utils.is_today(date, CAISO.default_timezone):
-            url = _BASE + "/%s.csv" % file
-            msg = f"Fetching URL: {url}"
-            log(msg, verbose)
-
+def _get_historical(
+    file: str,
+    date: str | pd.Timestamp,
+    verbose: bool = False,
+) -> pd.DataFrame:
+
+    if date == "today":
```

Comment:
it's possible `date` doesn't come through as the string today. I think you'll want to us `utils.is_today(date, CAISO.default_timezone)`

Reply:
Done